test/paraiso_test.exs

```diff
@@ -38,7 +40,11 @@ defmodule ParaisoTest do
           "email_address" => "keshihoriuchi2@gmail.com",
           "is_primary" => false
         }
-      ]
+      ],
+      "options" => %{
```

Comment:
You already tested functionality in doctest. So you should choose more realistic values in this testcase for testing more usecase level requirements.

Reply:
Remove test values.